### PR TITLE
add doc for contributions

### PR DESCRIPTION
**Pull request by antonioortegajr**
add doc for contributions steps

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. [My stackoverflow question](http://stackoverflow.com/q/12488905/180626) includes sample code that *anyone* with a base R setup can run to reproduce what I was seeing
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
```
Comment (CesarMMGarcia) [inline CONTRIBUTING.md:37]: screenshots are also very helpful

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
```
Comment (CesarMMGarcia) [inline CONTRIBUTING.md:11]: We use github to host code, track issues and feature requests, as well as accept pull requests.

quick edit, remove the second 'to'.

**Review by CesarMMGarcia — APPROVED** [review]
quick lil changes but looks good.

File: CONTRIBUTING.md (hunk 1 above)
Comment (antonioortegajr) [reply CONTRIBUTING.md:37]: added line

**Pull request merged**